react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+ReactDOM.render(CustomButton(), document.getElementById('root'));
```

Comment:
With components we don't want to call them like a normal function. You use them as a JSX element like: `<CustomButton />`